bazel/protobuf.patch

```diff
@@ -1,33 +1,8 @@
-diff --git a/src/google/protobuf/stubs/strutil.cc b/src/google/protobuf/stubs/strutil.cc
-index 3844fa6b8b..5486887295 100644
---- a/src/google/protobuf/stubs/strutil.cc
-+++ b/src/google/protobuf/stubs/strutil.cc
-@@ -1065,10 +1065,12 @@ char* FastUInt32ToBufferLeft(uint32 u, char* buffer) {
- }
- 
- char* FastInt32ToBufferLeft(int32 i, char* buffer) {
--  uint32 u = i;
-+  uint32 u = 0;
-   if (i < 0) {
-     *buffer++ = '-';
--    u = -i;
-+    u -= i;
-+  } else {
-+    u = i;
-   }
-   return FastUInt32ToBufferLeft(u, buffer);
- }
-
-diff --git a/BUILD b/BUILD
-index 6665de94..55f28582 100644
---- a/BUILD
-+++ b/BUILD
-@@ -19,7 +19,7 @@ config_setting(
- # ZLIB configuration
- ################################################################################
- 
--ZLIB_DEPS = ["@zlib//:zlib"]
-+ZLIB_DEPS = ["//external:zlib"]
- 
- ################################################################################
- # Protobuf Runtime Library
+# https://github.com/protocolbuffers/protobuf/pull/6720
+diff --git a/third_party/BUILD b/third_party/BUILD
+new file mode 100644
+index 0000000000..b66101a39a
+--- /dev/null
++++ b/third_party/BUILD
+@@ -0,0 +1 @@
++exports_files(["six.BUILD", "zlib.BUILD"])
```

Comment:
Don't we have our own zlib above?

Reply:
this is a cherry-pick of https://github.com/protocolbuffers/protobuf/pull/6720 (commented above)